build.sbt

```diff
@@ -12,6 +12,22 @@ Global / onChangedBuildSource := ReloadOnSourceChanges
 
 ThisBuild / turbo := true
 
+val customScalaJSVersion = Option(System.getenv("SCALAJS_VERSION"))
+
+inThisBuild(List(
+  organization := "org.typelevel",
+  homepage := Some(url("http://www.squants.com/")),
+  licenses := Seq("Apache 2.0" -> url("http://www.opensource.org/licenses/Apache-2.0")),
+  developers := List(
+    Developer(
```

Comment:
At this point, I don't think I should be listed as the only dev.  Others have been contributing so much more.  I'd say we should either add others to this list or perhaps drop it all together (which I think is the general recommendation).

Reply:
Ok I may do that in another PR which can be used to verify the release process